mani_skill/utils/download_asset.py

```diff
@@ -186,6 +186,11 @@ def initialize_extra_sources():
         url="https://github.com/haosulab/ManiSkill-UnitreeH1/archive/refs/tags/v0.1.0.zip",
         target_path="robots/unitree_h1",
     )
+    DATA_SOURCES["unitree_g1"] = DataSource(
+        source_type="robot",
+        url="https://github.com/matheecs/ManiSkill-UnitreeG1/archive/refs/tags/v0.1.0.zip",
```

Comment:
I have forked this to our own repository. Can you change the link to `https://github.com/haosulab/ManiSkill-UnitreeG1/archive/refs/tags/v0.1.0.zip`

Reply:
Sure. Please pull the latest update from the original repository (with the new simplified model added `g1_simplified.urdf`).